dts/common/nordic/nrf54h20_eng0.dtsi

```diff
@@ -63,6 +63,12 @@
 			write-block-size = <16>;
 		};
 
+		ficr: memory@fffe000 {
+			compatible = "nordic,nrf-ficr";
+			reg = <0xfffe000 0x800>;
+			#ficr-cells = <1>;
+		};
```

Comment:
nit: Maybe move under `cpurad_uicr`, to keep the nodes sorted by unit address. Could use `DT_SIZE_K(2)` as well.

Reply:
done